README.md

```diff
@@ -1,9 +1,14 @@
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://github.com/esciencecenter-digital-skills/NEBULA/blob/main/LICENSE)
+
 # NEBULA
 
 The NEBULA (Nuxt & eScience Based Universal Learning Application) framework can be used to create an easily maintainable, version-controllable, web-based lesson collection. It is based on [Nuxt.js](https://nuxtjs.org/) and [Vue.js](https://vuejs.org/).
 
 NEBULA is being created as part of the Netherlands eScience Center Digital Skills programme.
 
+Copyright 2023 Netherlands eScience Center
```

Comment:
Ah sorry, missed this comment.

Yes we do need a year, because copyright expires a certain number of years after the first time the work is published, and so the year of first publication needs to be added so that we know when the work enters the public domain.

If different parts of the work were first published in different years, then you need to list all of them. I guess updating would be correct if you release a new version that completely replaces everything from the previous release, but that's very rare I guess. If something is developed continuously, then you can write 2023-2026 to shorten things a bit.

See also https://www.gnu.org/licenses/gpl-howto.en.html, under "The copyright notice"